File: sentimental-cash/cash.py
from cs50 import get_float


def get_positive_float(prompt):
    while True:
        try:
            value = float(input(prompt))
            if value > 0:
                return value
        except ValueError:
            pass


def main():

    money = get_positive_float("Change: ") * 100

    count_25 = money // 25
    money %= 25

    count_10 = money // 10
    money %= 10

    count_5 = money // 5
    money %= 5

    count_1 = money // 1
    money %= 1

    print(int(count_25 + count_10 + count_5 + count_1))


if __name__ == "__main__":
    main()

File: cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int money = get_int("Change owed: ");
    while (money <= 0)
    {
        money = get_int("Change owed: ");
    }
    int count_25 = money / 25;
    int count_10 = (money - count_25 * 25) / 10;
    int count_5 = (money - count_25 * 25 - count_10 * 10) / 5;
    int count_1 = (money - count_25 * 25 - count_10 * 10 - count_5 * 5) / 1;
    printf("%i\n", count_25 + count_10 + count_5 + count_1);
}
